examples/acquiring_data/downloading_lascoC2.py

```diff
@@ -1,41 +0,0 @@
-"""
```

Comment:
👍 

I think back in the day we had this due to the metadata needing to be manually corrected.